src/commands.c

```diff
@@ -88,6 +88,10 @@ _Bool fp_warp(u16 area, u16 map, u16 entrance) {
         pm_state_step_end_battle();
     }
 
+    pm_func_800554A4(0); // stop koopa village radio from playing
+    pm_func_800554A4(1);
+    pm_func_800554A4(2);
+    pm_func_800554A4(3);
     pm_PlayAmbientSounds(-1, 0);   // clear ambient sounds
```

Comment:
this line is now unnecessary since `pm_func_800554A4(0)` duplicates this functionality. honestly I'd just get rid of the whole function since we most likely won't need it again, and the signature doesn't quite match decomp anymore anyway. otherwise everything looks good!